examples/blog/src/components/HeaderLink.astro

```diff
@@ -1,5 +1,9 @@
 ---
-export interface Props extends astroHTML.JSX.AnchorHTMLAttributes {}
+import type { HTMLAttributes } from 'astro/types';
+
+interface Props extends HTMLAttributes<'a'> {
+	class?: string;
```

Comment:
Is this line needed? Presumably, `HTMLAttributes<'a'>` already includes a definition for `class`.